src/main/java/se/sundsvall/contactsettings/integration/db/ContactSettingRepository.java

```diff
@@ -0,0 +1,22 @@
+package se.sundsvall.contactsettings.integration.db;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.CrudRepository;
+
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+import se.sundsvall.contactsettings.integration.db.model.ContactSettingEntity;
+
+@CircuitBreaker(name = "contactSettingRepository")
+public interface ContactSettingRepository extends CrudRepository<ContactSettingEntity, String>, JpaRepository<ContactSettingEntity, String>, JpaSpecificationExecutor<ContactSettingEntity> {
```

Comment:
Missing a way to find by destination. Should return list of ContactSettingEntity